CODEOWNERS

```diff
@@ -1 +1,2 @@
 /examples/ @addisonschultz
+/examples/@miroapp/devx
```

Comment:
You can simplify this 

```
/examples/ @addisonschultz @miroapp/devx
```

Or if you want to keep it the same you need to add a space

```diff
/examples/ @addisonschultz
- /examples/@miroapp/devx
+ /examples/ @miroapp/devx
```